Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="jacktyler2391@gmail.com"
+
+ENV PYTHONBUFFERED 1
```

Comment:
There is a typo in the `ENV` instruction. It should be `PYTHONUNBUFFERED` instead of `PYTHONBUFFERED` to ensure that Python output is sent straight to the terminal without being buffered.